taxi/urls.py

```diff
@@ -0,0 +1,10 @@
+from django.urls import path
+
+from .views import index
+
+
+urlpatterns = [
+    path("", index, name="index"),
+]
+
+app_name = "taxi"

```

Comment:
The `app_name` declaration is typically placed at the top of the file, before the `urlpatterns` list. Consider moving it to the top for better readability and convention.